event-sourcing/event-repository/event-repository-jdbc/src/main/java/uk/gov/justice/services/eventsourcing/repository/jdbc/eventstream/EventStreamJdbcRepository.java

```diff
@@ -48,20 +50,25 @@ public void insert(final EventStream eventStream) {
     }
 
     @Override
-    public Stream<EventStream> getFeed(final long offset, final Link link,
-                                       final long pageSize, final Map<String, Object> params) {
+    public Stream<EventStream> getFeed(final String position,
+                                       final Direction direction,
+                                       final long pageSize,
+                                       final Map<String, Object> params) {
         try {
             final PreparedStatementWrapper ps = preparedStatementWrapperOf(SQL_PAGE);
             ps.setLong(1, pageSize);
-            ps.setLong(2, offset);
+            if (FixedPosition.HEAD.getPosition().equals(position)){
+
+            }
+            ps.setLong(2, valueOf(position));
             return streamOf(ps);
         } catch (SQLException e) {
             throw new JdbcRepositoryException(READING_STREAM_EXCEPTION, e);
         }
     }
 
     @Override
-    public boolean recordExists(final long offset, final Map<String, Object> params) {
+    public boolean recordExists(long offset, Map<String, Object> params) {
```

Comment:
Final is missing, but don't fix it... this is just for streams (new branch)

Reply:
Will be refactored in the next commit